examples/lighting-app/nrfconnect/main/ZclCallbacks.cpp

```diff
@@ -70,5 +72,17 @@ void MatterPostAttributeChangeCallback(const chip::app::ConcreteAttributePath &
  */
 void emberAfOnOffClusterInitCallback(EndpointId endpoint)
 {
+    EmberAfStatus status;
+    bool storedValue;
+
+    // Read storedValue on/off value
+    status = Attributes::OnOff::Get(1, &storedValue);
+    if (status == EMBER_ZCL_STATUS_SUCCESS)
+    {
+        // Set actual state to stored before reboot
```

Comment:
What does this have to do with reboot?

Reply:
Maybe this is trying to say "set actual state to the state of the cluster that was last persisted" or something?